### PR TITLE
Moonbeam related wallet changes

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/224
https://github.com/PureStake/moonbeam-docs/pull/213/files
https://github.com/PureStake/moonbeam-docs/pull/231

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

https://github.com/PureStake/moonbeam-docs/pull/233

This needs to be merged at the same time that Moonbeam Doc 233 PR is merged.

### CODE REVIEW

**Review by eshaben — CHANGES_REQUESTED** [review]

File: builders/get-started/moonbeam.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ description: 学习如何通过RPC和WSS端点快速连接至波卡（Polkadot
 由于Moonbeam尚未启用EVM功能，目前暂不能连接至MetaMask。Moonbeam网络完全启动后，您可遵循相关教程连接MetaMask至Moonbeam。
 
 <!---
```
[the remark this answers is not in the document]
Comment (eshaben) [reply builders/get-started/moonbeam.md:25]: ah saw you made this change in another PR so nevermind this comment 👍

**Comment by hyd628** [comment]
Thanks @eshaben all feedbacks have been incorporated in the above commit.

**Review by eshaben — APPROVED** [review]
Awesome! LGTM ✨

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**